lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -124,11 +112,26 @@ public function getExtension()
      */
     protected function getFilename($id)
     {
+        if ('\\' === DIRECTORY_SEPARATOR) {
```

Comment:
So what do you suggest to change? Can we detect case-insensitive fs in a sane manner? We could create a temporary file once in the constructor and check if we can access it case-insensitive. Do you prefer that?